frontend/apps/reader/readerui.lua

```diff
@@ -447,7 +456,7 @@ function ReaderUI:showFileManager(file)
         last_dir, last_file = util.splitFilePathName(file)
         last_dir = last_dir:match("(.*)/")
     else
-        last_dir, last_file = self:getLastDirFile()
+        last_dir, last_file = self:getLastDirFile(true)
```

Comment:
Either you remove the `true` here (as it's not expected and managed by `ReaderUI:getLastDirFile()` above.
Or you have it manage it as in my sample code:
```lua
+ function ReaderUI:getLastDirFile(to_file_browser) 
+    if to_file_browser and self.last_dir_for_file_browser then
+        return self.last_dir_for_file_browser
+    end
```

(It's possible `ReaderUI:getLastDirFile()` might not be used in other context, but I guess it makes sense to trigger that explicitely with a `to_file_browser=true` parameter)

Reply:
I overlooked that function parameter. 
That's why it didn't want to work at first time!;)